test/runtests.jl

```diff
@@ -260,3 +260,24 @@ end
 @testset "slow comparisons" begin
     @test iszero(@allocated(3.0 <= invsqrt2))
 end
+
+# issues #43
+@testset "macro error" begin
+    err_type = VERSION < v"1.3" ? LoadError : ArgumentError
+    @test_throws err_type macroexpand(@__MODULE__, :(IrrationalConstants.@irrational Myπ big(π) Myπ))
```

Comment:
IMO it's easier to use `@macroexpand`, e.g.
```suggestion
    @test_throws err_type @macroexpand(IrrationalConstants.@irrational Myπ big(π) Myπ)
```